welcome/index.adoc

```diff
@@ -81,17 +81,17 @@ You can also deploy a cluster on AWS infrastructure that you provisioned yoursel
 ifndef::openshift-origin[]
 - **xref:../installing/installing_ibm_cloud_public/preparing-to-install-on-ibm-cloud.adoc#preparing-to-install-on-ibm-cloud[Install a cluster on IBM Cloud VPC]**: You can install {product-title} on IBM Cloud VPC on installer-provisioned infrastructure.
```

Comment:
I might not understand all the nuances but should this be "IBM Cloud(R)" by using the new variable {ibm-cloud-name} and not "IBM Cloud VPC"?

Reply:
Excellent catch. I missed this. Updated.